api/app/controllers/webhook.controller.js

```diff
@@ -73,6 +72,12 @@ exports.create = async (req, res) => {
   const html_url = pull_request.pull_request?.html_url;
   const currentTime = new Date();
 
+  if (!action || !html_url) {
```

Comment:
This has been taken care of by lines 64-68 as the entire payload from the webhook is a PR event we tick, it won't send to any listener if the data is not complete, listening for any other action, I don't think it's necessary except explicitly defined as a requirement. Initially don't want our endpoint to be listening for pings when testing. Though still doesn't have a side effect.

Reply:
That makes sense. I added this for the ping since the 503 error was confusing at first. I can remove it if you want

Reply:
You can leave it. It's an extra catch. Wasn't thinking it's necessary.